multitemporal/mt.py

```diff
@@ -365,10 +383,15 @@ def run(projdir, outdir, projname, sources, steps,
                 prog += 1
                 print('mt {:0.02f} complete.\r'.format(pct))
             results.append(r)
-    else:
+    elif nproc == 1:
         results = []
         for job in jobs:
             results.append(func(job))
+    elif nproc < 0:
+        # secret way to test
+        results = [func(jobs[-nproc-1])]
+    else:
+        raise Exception("nproc can't be zero")
```

Comment:
This isn't `Exception`, this is `ValueError`.  Never raise Exception; either pick a subtype or make a new one.